Dockerfile

```diff
@@ -39,6 +39,8 @@ RUN go generate
 # Build the Go app
 RUN go build -o /ferrum
 
+RUN go build -o ferrum-admin.exe ./api/admin/cli
```

Comment:
seems that name should be `ferrum-admin` without `.exe`

Reply:
Ready